src/content/menus/footer.yaml

```diff
@@ -31,6 +31,8 @@ sections:
         link: /blog/
       - name: Newsletter
         link: https://weekly.nixos.org/
+      - name: Research and scientific publications
```

Comment:
This link now looks pretty long in the footer, maybe we should come up with a shorter title

Reply:
Maybe. I think 'Research publications' would be good, 'Scientific publications' would be OK as well.
I do like the current title the best, but if it does not look good I am fine with shortening it.

I do think that title of the page itself should stay the same though, and I am not sure we should change it in the footer anyways or not.